.github/ISSUE_TEMPLATE/epic.yml

```diff
@@ -0,0 +1,104 @@
+    description: TLDR description of the epic. Give a succinct and plain overview of what the epic is about.
+    placeholder: Give a succinct and plain overview of what the epic is about.
+  validations:
+    required: true
+- type: textarea
+  id: context
+  attributes:
+    label: Context
+    description: What thinking led to this? Provide any necessary historical context required to understand this epic.
+    placeholder: Provide any necessary historical context required to understand this epic.
+  validations:
+    required: true
+- type: textarea
+  id: value
+  attributes:
+    label: Value
+    description: Why should we do it? How do we know this is a real problem and worth solving?
+    placeholder: Explicitly describe the value to Gitpod and/or our users. I.e. why answer should we undertake this epic?
+  validations:
+    required: true
+- type: textarea
+  id: acceptance-criteria
+  attributes:
+    label: Acceptance Criteria
+    description: What needs to be done before the work is considered complete? The checks which must be complete for this epic to be considered done.
+    placeholder: Defines clearly when the work is complete. Acts as a litmus test for "done" and avoids "done" being ambiguous. Useful for implicit assumptions, e.g. ensuring docs updates are not forgotten.
+  validations:
+    required: true
+- type: textarea
+  id: measurement
+  attributes:
+    label: Measurement
+    description: How will we know whether we've been successful / solved the problem? How will you measure the success of the epic? Ideally this metric is one of our key product metrics.
+    placeholder: Important as it's how we track the outcomes (not just output) of the work and prove a change was worth it. Or it should be removed or iterated.
+  validations:
+    required: true
+- type: textarea
+  id: growth-area
+  attributes:
+    label: Growth Area
+    description: Which aspect of Gitpod do we expect improvements in? Acquisition/Onboarding/Exploration/Expansion as defined in [Funnel Proposal](https://www.notion.so/gitpod/Funnel-Proposal-d7d0dba8aced4184b660092a74f8dd3a) (internal)
+    placeholder: Growth is key. This allows us to frame epics from a growth context. Which areas are we expecting this epic to help us with our growth initiatives?
+  validations:
+    required: false
+- type: textarea
+  id: personas
+  attributes:
+    label: Persona(s)
+    description: Who will be impacted by this change? Which of our personas will be impacted by this change?
+    placeholder: Why? To bring persona's into our work. Persona's can help us prioritise our markets. Currently, we are not focusing on the education/training persona currently. We should avoid epics which target this persona.
+  validations:
+    required: false
+- type: textarea
+  id: hypothesis
+  attributes:
+    label: Hypothesis
+    description: If we do X, we expect Y
+    placeholder: Can be useful if the work is explicitly experimental.
+  validations:
+    required: false
```

Comment:
Suggest removing, and merging with measurement [[1](https://github.com/gitpod-io/gitpod/pull/7556/files#r838382804)]

```suggestion
```